README.md

```diff
@@ -20,6 +20,8 @@ Status of OpenCord
 - [ ] Plugin support
 - [ ] Multiplatform support (Windows, macOS, Linux, Android)
 
+#### for more information see [status](/STATUS.md)
+
```

Comment:
This section could be moved as a subsection of `Full`

Reply:
I don't think so, since STATUS.md also contains info about multiplatform, plugins and themes